modules/nextflow/src/main/groovy/nextflow/scm/AssetManager.groovy

```diff
@@ -737,30 +793,32 @@ class AssetManager {
     /**
      * @return A list of existing branches and tags names. For example
      * <pre>
-     *     * master (default)
-     *       patch-x
-     *       v1.0 (t)
-     *       v1.1 (t)
+     *     * P master (default)
+     *         patch-x
+     *         v1.0 (t)
+     *         v1.1 (t)
      * </pre>
      *
-     * The star character on the left highlight the current revision, the string {@code (default)}
-     *  ticks that it is the default working branch (usually the master branch), while the string {@code (t)}
-     *  shows that the revision is a git tag (instead of a branch)
+     * The star character on the left highlights the selected revision,
```

Comment:
I think the star does not have any more sense because in this model there isn't anymore an "active" revision

Reply:
thanks, let's review this after the design points below are sorted out

Reply:
ok, on this one now

Reply:
OK, done:
```
$ nextflow info hello
 project name: nextflow-io/hello
 repository  : https://github.com/nextflow-io/hello
 local path  : /home/ubuntu/add-worktree/HOME_PR/assets/nextflow-io/hello
 main script : main.nf
 revisions   : 
 P master (default)
   mybranch
   testing
 P toy
   v1.1 [t]
 P v1.2 [t]
```

`nextflow info <project` now:
- does not take a revision parameter
- always show `local path` for the default revision (also documented in the docs)
- does not show the sticky `*` for current revision, instead shows `P` for locally pulled revisions

@pditommaso